locales-hi-IN.xml

```diff
@@ -0,0 +1,361 @@
+<?xml version="1.0" encoding="utf-8"?>
+<locale xmlns="http://purl.org/net/xbiblio/csl" version="1.0" xml:lang="hi-IN">
+  <info>
+    <translator>
+      <name>Neha Srivastava</name>
+      <email>neha-srivastava@outlook.com</email>
+    </translator>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <updated>2021-02-22T12:13:02+05:30</updated>
+  </info>
+  <style-options punctuation-in-quote="true"/>
+  <date form="text">
+    <date-part name="day" suffix=" "/>
+    <date-part name="month" suffix=", "/>
+    <date-part name="year"/>
+  </date>
+  <date form="numeric">
+    <date-part name="day" form="numeric-leading-zeros" suffix="/"/>
+    <date-part name="month" form="numeric-leading-zeros" suffix="/"/>
+    <date-part name="year"/>
+  </date>
+  <terms>
+    <term name="accessed">अंतिम उपयोग</term>
+    <term name="and">व</term>
+    <term name="and others">व अन्य</term>
+    <term name="anonymous">अज्ञात</term>
+    <term name="anonymous" form="short">अज्ञात</term>
+    <term name="at">पर</term>
+    <term name="available at">पर उपलब्ध</term>
+    <term name="by">द्वारा</term>
+    <term name="circa">सन </term>
+    <term name="circa" form="short">सन.</term>
+    <term name="cited">उल्लेखित</term>
+    <term name="edition">
+      <single>संस्करण</single>
+      <multiple>संस्करण</multiple>
+    </term>
+    <term name="edition" form="short">ed.</term>
+    <term name="et-al">इत्यादि</term>
+    <term name="forthcoming">आगामी</term>
+    <term name="from">से</term>
+    <term name="ibid">उक्त</term>
+    <term name="in">में</term>
+    <term name="in press">मुद्रण में</term>
+    <term name="internet">इंटर्नेट</term>
+    <term name="interview">साक्षात्कार</term>
+    <term name="letter">पत्र</term>
+    <term name="no date">दिनांक अज्ञात</term>
+    <term name="no date" form="short">n.d.</term>
+    <term name="online">ऑनलाइन</term>
+    <term name="presented at">पर प्रस्तुत</term>
+    <term name="reference">
+      <single>संदर्भ</single>
+      <multiple>references</multiple>
+    </term>
+    <term name="reference" form="short">
+      <single>ref.</single>
+      <multiple>refs.</multiple>
+    </term>
+    <term name="retrieved">पुनर्प्राप्त</term>
+    <term name="scale">scale</term>
+    <term name="version">संस्करण</term>
+
+    <!-- ANNO DOMINI; BEFORE CHRIST -->
+    <term name="ad">CE</term>
+    <term name="bc">BCE</term>
+
+    <!-- PUNCTUATION -->
+    <term name="open-quote">“</term>
+    <term name="close-quote">”</term>
+    <term name="open-inner-quote">‘</term>
+    <term name="close-inner-quote">’</term>
+    <term name="page-range-delimiter">–</term>
+
+    <!-- ORDINALS -->
+    <term name="ordinal">वाँ</term>
+    <term name="ordinal-01"></term>
+    <term name="ordinal-02"></term>
+    <term name="ordinal-03"></term>
+    <term name="ordinal-04"></term>
+    <term name="ordinal-06"></term>
```

Comment:
@neha-sri, can you explain how ordinal suffixes work in Hindi? (the "st" in "1st" and "nd" in "2nd")

Is "वाँ" always the suffix? If so, we should delete the other "ordinal-0x" terms here.

Reply:
So that part is complicated. Entire words change for 1st, 2nd, 3rd, 4th and 6th. वाँ doesnt work as an ordinal suffix for these numbers but then neither does any other suffix. I'll write a few so you'll get it. 

1st = पहला (péhlā) 
2nd = दूसरा (d̪oosrā)
3rd = तीसरा (t̪eesrā)
4th = चौथा (chaut̪hā)
5th = पाँचवाँ (pānchvān)
6th = छठा (chhat̪hhā)
7th = सातवाँ (sātvān)
8th = आठवाँ (āthvān)
9th = नवाँ (navān) 
10th = दसवाँ (dusvān) and so on.. 

So the ones with the वाँ can be abbreviated as 7वाँ but the others cannot be abbreviated as 1ला for example. Hence, this solution.

Reply:
Okay, let's leave it as is, then.